src/ServicePulse.Host/vue/src/components/failedmessages/AllDeletedMessages.vue

```diff
@@ -57,18 +61,13 @@ function loadMessages() {
   return loadPagedMessages(groupId.value, pageNumber.value, "", "", startDate.toISOString(), endDate.toISOString());
 }
 
-async function loadGroupDetails(groupId) {
+async function loadGroupDetails(groupId: string) {
   const response = await useFetchFromServiceControl(`archive/groups/id/${groupId}`);
```

Comment:
useTypedFetchFromServiceControl? Although mostly throwaway, it would enable us to specify what we expect to be returned from this endpoint, and will also enable consistent error handling